sim/sim.go

```diff
@@ -33,12 +33,12 @@ func NewSimulation(simConfig Config, graniteConfig f3.GraniteConfig, traceLevel
 	vrf := f3.NewVRF(ntwk)
 
 	// Create participants.
-	genesisPower := f3.NewPowerTable()
+	genesisPower := f3.NewPowerTable(make([]f3.PowerEntry, 0))
 	participants := make([]*f3.Participant, simConfig.HonestCount)
 	for i := 0; i < len(participants); i++ {
 		participants[i] = f3.NewParticipant(f3.ActorID(i), graniteConfig, ntwk, vrf)
 		ntwk.AddParticipant(participants[i])
-		genesisPower.Add(participants[i].ID(), 1)
+		genesisPower.Add(participants[i].ID(), f3.NewStoragePower(1), make([]byte, 0))
```

Comment:
Let's put a non-empty key in here so that the fake signature verification at least checks it's the right signature. Serialize the participant ID into a byte array, and maybe add some magic bytes to distinguish it from other serialized integers.

Reply:
Done.